### DISCUSSION TITLE
Custom Shortener

### DISCUSSION THREAD

**Discussion by RayHollister** [asdfdotdev/utm.codes#54] [Q&A] [answered]
Do you have any suggestions for a custom shortener that works well with utm.codes? I tried Hum, but I need one that allows us to create short URLs manually to a few other domains that we run.

**Comment by chrislarrycarl** [comment, marked as answer]
utm.codes comes with support for both [bitly](https://bitly.com/) and [rebrandly](https://www.rebrandly.com/), both very good services that provide a free tier so you can try them out risk-free (which is a big reason why they're included by default 😄).

If you find a shortener that works better for your needs you can also [add support for it](https://github.com/asdfdotdev/utm.codes/wiki/Custom-Shortener) fairly easily - and if they have a free tier [let me know](https://github.com/asdfdotdev/utm.codes/issues) and I'll see about adding them by default as well.

**Reply by RayHollister** [reply]
Thanks. I'm giving a shot at using redirection.me. We already use it on our sites, and it has a decent API. It isn't specifically for short URLs, but it does a better job than most and it uses our domain name. I'll let you know once I've tested it.
